docusaurus/docs/publish-a-plugin/publish-or-update-a-plugin.md

```diff
@@ -37,7 +37,7 @@ Follow these steps to publish your plugin for the first time.
 1. In the left menu, under Org Settings, click **My Plugins**.
 1. Click **Submit New Plugin**. The Create Plugin Submission dialog box appears.
 
-   ![Submit plugin.](/img/plugins-submission-create2.png)
+   ![Submit plugin.](/img/plugins-submission-create.png)
```

Comment:
The image returns a 404. The correct URL is:

```suggestion
   ![Submit plugin.](/img/plugin-submission-create.png)
```

Also the old image (`img/plugins-submission-create2.png`) is not used anymore and should be deleted

Reply:
think this is fixed now?

Reply:
Yep the link has been fixed :raised_hands: 

However the old image `img/plugins-submission-create2.png` should be deleted.